lib/Rest/VstRequest.cpp

```diff
@@ -113,15 +113,24 @@ void VstRequest::setHeader(VPackSlice keySlice, VPackSlice valSlice) {
   std::string value = valSlice.copyString();
   std::string key = StringUtils::tolower(keySlice.copyString());
   std::string val = StringUtils::tolower(value);
-  static const char* mime = "application/json";
-  if (key == StaticStrings::Accept && val.length() >= 16 &&
-      memcmp(val.data(), mime, 16) == 0) {
+  if (key == StaticStrings::Accept &&
+      val.length() == StaticStrings::MimeTypeJsonNoEncoding.length() &&
```

Comment:
the previous version checked if the *prefix* of the header value was `application/json`.
the new code checks if the complete header value is `application/json`. 
Wouldn't it be better to use a prefix check here?